README.md

```diff
@@ -1,31 +1,10 @@
-# [Stardoc](https://skydoc.bazel.build) - Starlark Documentation Generator
+# [Skydoc](https://skydoc.bazel.build) - Skydoc Documentation Generator
 
 [![Build status](https://badge.buildkite.com/c793ed98a3802393dba9813f524bfe28a87da2ab235df38c49.svg)](https://buildkite.com/bazel/skydoc-postsubmit)
 
-Stardoc is a documentation generator for [Bazel](https://bazel.build) build rules
+Skydoc is a **heavily deprecated** Python tool to generate documentation for [Bazel](https://bazel.build) build rules
```

Comment:
add "this repo is no longer maintained?"

Reply:
Good idea. Done.